pkg/controller/util/placement.go

```diff
@@ -24,11 +24,17 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
-type GenericPlacementSpec struct {
+type GenericPlacementFields struct {
 	ClusterNames    []string              `json:"clusterNames,omitempty"`
 	ClusterSelector *metav1.LabelSelector `json:"clusterSelector"`
```

Comment:
(suggestion) Shouldn't we use `omitempty` json tag for this field as well?

Reply:
Done